reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -64,8 +68,8 @@ PowerShell 7 installs to a new directory, enabling side-by-side execution with W
 Install locations by version:
 
 - Windows PowerShell 5.1: `C:\Windows\System32\WindowsPowerShell\v1.0`
-- PowerShell Core 6.x: `%programfiles%\PowerShell\6`
-- PowerShell 7: `%programfiles%\PowerShell\7`
+- PowerShell Core 6.x: `$Env:programfiles\PowerShell\6`
```

Comment:
NIT, but the casing should be: $env:ProgramFiles

Reply:
Fixed